config/locales/en-GB.yml

```diff
@@ -100,6 +100,8 @@ en-GB:
   powered_by: Powered by
   blocked_cookies_alert: "Your browser may be blocking cookies needed to use this shopfront. Click below to allow cookies and reload the page."
   allow_cookies: "Allow Cookies"
+  actions:
+    create_and_add_another: "Create and Add Another"
```

Comment:
you shouldn't change any locale file other than `en.yml`. That one is the master and then we translate the new keys using transifex.com. There are few guidelines written at the top of https://github.com/openfoodfoundation/openfoodnetwork/blob/master/config/locales/en.yml

Reply:
Hi @sauloperez Tks for your review. Actually I didn't know about the transifex service.. looks like is very handy. Anyway I'll remove the changes I made on the others YML files other than `en.yml`. 👍 😄

Reply:
it's one of those services that give you a UI on top of the yml's so that non-dev translators can provide the translation without messing with git.